.github/workflows/nodejs.yml

```diff
@@ -4,12 +4,20 @@ on: [push, pull_request]
 
 jobs:
   eslint:
+    name: Lint
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: hallee/eslint-action@master
```

Comment:
why remove? it adds comment to PRs

Reply:
the last linter i used (was java thou) actually did commit's and i kinda hated that
will switch back 😉